src/dsp_tools/utils/logger_config.py

```diff
@@ -0,0 +1,43 @@
+import sys
+from pathlib import Path
+
+from loguru import logger
+
+
+def logger_config() -> None:
+    """
+    This function configures the log files.
+    Currently, there are three sinks:
+    - logger.log in the home directory for development purposes
+    - warnings.log in the cwd only with level warning and higher for the user (no stack-trace)
+    - print output on the terminal, formatted the same as the warnings.log
+
+    Returns:
+        None
+    """
+    # If this is not removed, the default formatting is also printed out on the terminal
+    logger.remove()
+
+    text_format = "{time:YYYY-MM:DD HH:mm:ss.SSSS} | <level>{level} | {message}</level>"
```

Comment:
```suggestion
    text_format = "{time:YYYY-MM-DD HH:mm:ss.SSS} | <level>{level} | {message}</level>"
```

Reply:
**thought**: In the old logging, I found it useful to have the file name and line number, to know where the message was emitted.

Reply:
The line and filename are included in the logs